tests/performance_tests/performance_test_basic.py

```diff
@@ -580,6 +591,8 @@ def createArgumentParser():
     def parseArgs():
         ptbParser=PtbArgumentsHandler.createArgumentParser()
         args=ptbParser.parse_args()
+        if args.contracts_console:
+            print("Enabling contracts-console will not print anything unless debug level is 'debug' or higher.")
```

Comment:
Would it be worth checking and/or printing out the configured debug level in this message and the cli argument to change it?